luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/DefaultDimensionFactory.java

```diff
@@ -0,0 +1,118 @@
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.Factory;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.data.dimension.Dimension;
+import com.yahoo.bard.webservice.data.dimension.DimensionField;
+import com.yahoo.bard.webservice.data.dimension.DimensionRow;
+import com.yahoo.bard.webservice.data.dimension.SearchProvider;
+import com.yahoo.bard.webservice.data.dimension.metadata.StorageStrategy;
+import org.joda.time.DateTime;
+
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class DefaultDimensionFactory implements Factory<Dimension> {
+
+    @Override
+    public Dimension build(String name, ObjectNode configTable, LuthierIndustrialPark resourceFactories) {
+        return new Dimension() {
```

Comment:
Why are we building an anonymous inner class here? Why not an instance of `KeyValueStoreDimension`?